routes/owner.js

```diff
@@ -0,0 +1,101 @@
+const mongoose = require('mongoose')
+const express = require('express')
+const { Hotel, validate } = require('../models/hotel')
+const { Reservation } = require('../schemas/reservationSchema')
+const router = express.Router()
+
+// TODO: add auth middlewear, when it will be ready
+
+router.get('/hotels', async (req, res) => {
+  const hotel = await Hotel.find()
+
+  res.status(200).send(hotel)
+})
+
+router.post('/hotel', async (req, res) => {
+  const { error } = validate(req.body)
+  if (error) return res.status(400).send(error.details[0].message)
+
+  let hotel = new Hotel({
+    ownerId: req.body.ownerId,
+    localization: req.body.localization,
+    phoneNumber: req.body.phoneNumber,
+    name: req.body.name,
+    clientsRates: null,
+    email: req.body.email,
+    description: req.body.description,
+    rooms: [req.body.rooms],
+  })
+
+  await hotel.save()
+
+  res.status(200).send(hotel)
+})
+
+router.put('/hotel/:id', async (req, res) => {
+  const { error } = validate(req.body)
+  if (error) return res.status(400).send(error.details[0].message)
+
+  const hotel = await Hotel.findByIdAndUpdate(req.params.id, {
+    ownerId: req.body.ownerId,
+    localization: req.body.localization,
```

Comment:
Hmmm I think you can just pass req.body instead of passing all properties separately. It will automatically map through object properties and keys